build/generate-licenses.go

```diff
@@ -39,6 +39,14 @@ func main() {
 
 	defer util.Remove(file.Name())
 
+	if err := os.RemoveAll(destination); err != nil {
+		log.Fatalf("Cannot clean destination folder: %v", err)
+	}
```

Comment:
Personally, I am not a fan to do `RemoveAll` for a user directory. It may delete user files accidentally when a wrong path is used.

For example:
* https://github.com/go-gitea/gitea/issues/20191

If the flag option is "--options-dir=./options" and do `os.RemoveAll(optionsDir + "/license")`, it will be safer.

Just my option, not a blocker.

Reply:
TBH I only expect the CI to ever run this. Using a options directory is okay I guess, if there's an actual use-case someone might not want to use the license folder 🤷🏽

Reply:
If only CI needs this, then I think the option can be easily changed without breaking. And prevent some curious users from accidentally running it with a wrong option and deleting their files. 😁 

Otherwise L-G-T-M

Reply:
Feel free to make a *seperate* PR for it. Don't want to add more code than necessary for this PR and stay more on-topic.

Reply:
No problem. If others also agree to use RemoveAll, then no blocker from my side.